### ISSUE TITLE
Discussion: allow non-ASCII alphanumeric characters in filename slugs

### ISSUE THREAD

**Issue by Hunter-Github** [Phyks/BMC#29]
I know I'm opening a can'o'worms with this one (at least on Python 2) :(, yet sometimes there's a need to add a non-ASCII book/article to the library (Latin-1, Japanese, Cyrillic). I fiddled a bit with [`slugify`](https://github.com/Phyks/BMC/blob/master/libbmc/tools.py#L29-L44) in [libbmc/tools.py](https://github.com/Phyks/BMC/blob/master/libbmc/tools.py), but did not devote enough time. I'm leaving this featreq in hope of later coming back with a pull.

**Comment by Phyks** [comment]
True, but I do not have much examples of such papers in my library :/

**Comment by Phyks** [comment]
@Hunter-Github Could you give me an example of a case where such a thing would be useful? Thanks!
